aiDee/aiDee/BiometricType.swift

```diff
@@ -23,7 +23,9 @@ extension BiometricType {
             self = .touchID
         case .faceID:
             self = .faceID
-        default:
+        case .none:
```

Comment:
Why not just `default`? Saves the `self = .none` repetition.

Reply:
From Swift evolution 0192-non-exhaustive-enums which was implemented in `Swift 5`:

```
The downside of using a default case is that the compiler can no longer alert a developer that a particular enum has elements that aren't explicitly handled in the switch. To remedy this, switch cases will gain a new attribute, @unknown.
```
This is for enums that are not frozen. If Apple adds a new case you would have to be aware of it in order to update your code with the new case if we were to use a normal `default`. Here you will get a warning stating that not all cases are implemented: `Switch must be exhaustive`. 

If you click the fix me button the suggestion code by Apple is:
```
@unknown default:
            <#fatalError()#>
        }
```
So you could use `fatalError()` to produce a crash. However (although I am not a fan of duplication) I thought it would be better to just return `.none`

That was the reasoning behind the change. Happy to change back if required.